integration-tests/steps.go

```diff
@@ -238,12 +238,12 @@ var happyPathSteps = []Step{
 			chain:      providerChainId,
 			unbondFrom: 0,
 			sender:     0,
-			amount:     11000000,
+			amount:     1000000,
```

Comment:
This amount was made smaller to ensure that non-faulty validators maintain more than 2/3 validating power below